index.html

```diff
@@ -4,9 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href=styles/index.css rel="stylesheet">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header class="header_weather">
+        <h1>Weather Report</h1>
+        <span id="headerCity" class="header_city">
+            <!-- For the City of -->
+        </span>
+    </header>
+    <section class="city_section">
+        <h2>For the City of</h2>
+            <h2 id="city">Toms River</h2>
+            <input type="text" id="inputCity" value="Toms River">
+            <button id="reset">Reset</button>
+    </section>
+    <section class="temperature_section">
+        <h2>Temperature</h2>
+    <div class="temperature_content">
+        <div class="temperature_buttons">
+        <span id="increaseTemp">⬆️</span>
+        <span id="temperatureValue" class="blue"> 58°f </span>
```

Comment:
Not a big deal, but you could leave this element empty and inject your initial state value on first page load. The same goes for lines 47 and 48!